site/SiteConstants.ts

```diff
@@ -0,0 +1,13 @@
+// See https://polyfill.io/v3/url-builder/ for a list of all supported features
+const polyfillFeatures = [
```

Comment:
Rationale for dropping `es6, fetch, URL`: These are fully supported by all browsers that support async functions, so if the user should use an older browser, nothing will work anyway.